test/integration/api/datasets.js

```diff
@@ -2030,6 +2030,209 @@ describe('datasets and entities', () => {
             }));
       }));
 
+      it('should ignore a saveto incorrrectly placed on a bind on a structural field', testService(async (service) => {
```

Comment:
shouldn't we reject such forms?

Reply:
this [line](https://github.com/getodk/central-backend/blob/83e453326c9aeded6297c1ef5af63aa0efc782b4/lib/data/schema.js#L146C8-L146C8) of the code looks for binds that explicitly include `type`, so a field that is explicitly not a structural field. 

i believe the parsing code works like that (filtering for just binds with a type) because of this [test](https://github.com/getodk/central-backend/blob/de41d8fa09d8c6d3cd6ed652703974f1cce645d2/test/unit/data/schema.js#L139C36-L179) for this [issue](https://github.com/getodk/central-backend/issues/147) where sometimes there _is_ a bind on a structural field, but for something that collect/enketo does, not central.

it would be more work (and might introduce other bugs) to find these other binds and check they don't have a saveto and reject if they do... 

- it doesn't seem like a big enough problem to justify the work/risk: if a person adds a saveto on a structural field and it gets totally ignored, nothing terrible happens. hopefully they notice the problem when the property isn't in the list of properties for the dataset. they'd also have to have done it manually, because pyxform _does_ throw an error about this.
- if a person adds a type and a saveto on a bind for a structural field, many other things about the rendering of the form itself breaks.